ImageNet/train/google_amoeba_net_d_tpu_tensorflow17.json

```diff
@@ -0,0 +1,19 @@
+{
+  "version": "v1.0",
+  "author": "Google Cloud TPU",
+  "authorEmail": "cloud-tpu-dawnbench@googlegroups.com",
+  "framework": "TensorFlow v1.7rc1",
+  "codeURL": "https://github.com/tensorflow/tpu/tree/master/models/experimental/amoeba_net",
+  "model": "AmoebaNet-D N6F256",
+  "hardware": "GCP n1-standard-2, Cloud TPU",
+  "costPerHour": 6.595,
+  "timestamp": "2018-04-13",
+  "misc": {
```

Comment:
Because this is a new architecture, please add a link to the paper under `misc`.

Reply:
Done